.circleci/config.base.yml

```diff
@@ -694,6 +695,7 @@ workflows:
               only:
                 - master
                 - graphqlschemae2e
+                - fix-5393
```

Comment:
@yuth I think you forgot to exclude this after your tests.

Reply:
I am still running the tests. I will remove this when the tests pass